recipes/wayland-protocols/all/test_package/test_package.c

```diff
@@ -0,0 +1,18 @@
+
+#include <stdio.h>
+#include <wayland-client.h>
+#include "xdg-shell-client-protocol.h"
+
+// TODO: Need help to write actual test here 
```

Comment:
If anyone can help to write a test here that uses it... or maybe use a different protocol